Core/tests/Unit/GrpcRequestWrapperTest.php

```diff
@@ -109,6 +109,34 @@ public function responseProvider()
         ];
     }
 
+    public function testRetryFunction()
+    {
+        $requestWrapper = new GrpcRequestWrapper();
+        $timesCalled = 0;
+        $requestWrapper->send(
+            function () use (&$timesCalled) {
+                if (1 === $timesCalled++) {
```

Comment:
This is ironic since it was my suggestion, but this would be more readable like this:
```suggestion
                if (2 === ++$timesCalled) {
```